pokete_classes/fightmap.py

```diff
@@ -334,20 +383,50 @@ def fight(self, player, enemy, figure, settings, invitems, fightitems,
         self.show()
         self.clean_up(player, enemy)
         fightitems.mvmap.balls_label_rechar(figure.pokes)
+        self.logging.info("[Fight][%s] Ended, %s(%s) won", info["type"],
+                          winner.name, "player" if winner.player else "enemy")
         return winner
 
 
 class FightItems:
-    """Contains all fns callable by an item in fight"""
+    """Contains all fns callable by an item in fight
+    ARGS:
+        _map: FightMap object
+        movemap: MoveMap object
+        figure: Figure object
+        ob_maps: Dict of all PlayMaps
+        logging: logging module
 
-    def __init__(self, _map, movemap, figure, ob_maps):
+    The methods that can actually be called in fight follow the follwing patern:
+        ARGS:
+            obj: The players Poke object
+            enem: The enemys Poke object
+            info: The info dict
+        RETURNS:
+            1: The continue the attack round
+            2: The win the game
```

Comment:
What is meant with `The continue the attack round`?

Maybe try replacing one `the`?

Reply:
It should have been a `to`